### DISCUSSION TITLE
CloudLog CAT issues

### DISCUSSION THREAD

**Discussion by 2E0SVB** [magicbug/Cloudlog#1438] [Q&A] [answered]
Hello all, CloudLog n00b here.

My CloudLog account is up and running without issue. I've used it to log some QSOs today - all good. I have installed the latest version of CloudLog CAT. OmniRig was already installed. I've created a read write key and populated both of the data fields with the necessary information. CloudLog CAT isn't syncing for some reason. I'm sure I have done something silly; please see the attached screenshots and see if you can suggest a way forward?

73,

John
2E0SVB
![CloudLog1](https://user-images.githubusercontent.com/102095640/159364872-cfbded08-658f-4bed-a0fd-16c991d16da8.PNG)
![CloudLog2](https://user-images.githubusercontent.com/102095640/159364879-0501f587-588e-4c42-9c52-415211da717a.PNG)
![CloudLog3](https://user-images.githubusercontent.com/102095640/159364883-ef9b99b6-c8c8-4ad3-bcf1-3ce07dcea643.PNG)

**Comment by dg9vh** [comment, marked as answer]
Try to remove Index.php at the end...

2E0SVB ***@***.***> schrieb am Mo., 21. März 2022, 22:14:

**Reply by 2E0SVB** [reply]
Perfect!

Danke sehr!

John

**Comment by dg9vh** [comment]
Made this also on my beginning and IT tooks be hours 😉

2E0SVB ***@***.***> schrieb am Mo., 21. März 2022, 22:23:

**Reply by 2E0SVB** [reply]
Ha! :) 

I have just tried to enter a test QSO and CloudLog isn't selecting the band or populating the frequency automatically. How do I make it do that please?

**Reply by 2E0SVB** [reply]
Anyone? Anyone? Bueller?

**Reply by magicbug** [reply]
In the QSO Box go to "Station" then Select the radio and save a QSO! 👍🏻

**Reply by magicbug** [reply]
See the screenshot below

![chrome_8m2HsYJpgM](https://user-images.githubusercontent.com/84308/160495118-2ebdb73c-8690-4070-b5cf-38844aed538b.png)

**Reply by magicbug** [reply]
Also documented here https://github.com/magicbug/Cloudlog/wiki/Radio-Interface

**Comment by 2E0SVB** [comment]
Perfect, thank you so much.

Kind Regards,

John
________________________________

**Comment by dg9vh** [comment]
You have to select the Radio weithin the logging Tab...

2E0SVB ***@***.***> schrieb am Mo., 21. März 2022, 22:27:
